TickSpec/TickSpec.fsproj

```diff
@@ -38,13 +43,11 @@ Describe behaviour in plain text using the Gherkin business language, i.e. Given
     <Compile Include="FeatureParser.fs" />
     <Compile Include="Feature.fs" />
     <Compile Include="TickSpec.fs" />
+    <None Include="..\LICENSE.txt" Pack="true" PackagePath="LICENSE.txt"/>
```

Comment:
better to put in a solution folder, no?

Reply:
I prefer to have it in root of the repository, so it is immediately visible. Moreover, I believe that it is expected by the tools.

Reply:
Sorry, I mean I would not add a License to a `.fsproj`, but include it in a `.sln`
Spotted the `PackagePath` now -nmm!